meetings/breakout_meetings/2019-01-14-project-progression.md

```diff
@@ -0,0 +1,104 @@
+* Myles: One of the things that we want is an engaged an successful CPC. We need to identify the projects that want to be involved an participate. What are the shared bits that make these the same, and identify the bits that make them successful. Who’re the stakeholders who will show up on day one and what does it look like to receive help from those people?
+* Jory: I agree we’re looking for engaged cross-project engagement. To me that doesn’t look like a project doesn’t have to be super involved to help or be active – especially if they don’t meet some definition that we’ve defined here.
+* Myles: The only difference for top level projects is that they’d be the voting members if something went to a vote.
+* Dave: We need definition on what makes a top level member, and the steps to become that.
+* Jory: We have to define what open governance means.
+* Dave: React - although they accept PRs, are all the members who work on it FB employees? If so that’s not open governance. If the requirements are that the committers are employees or contractors of one org, it’s not open.
+* Tierney: There’s a project that isn’t open, and wants to move into an open foundation - if we have guide rails for them they shouldn’t be blocked. If it was react, they should be able to move into a ‘react org’, and adopt an open governance if they wanted to. That’s a success path.
+* Myles: We’re talking about becoming a top level project.
+* Michael: Could you join as a BDFL and join, and stay that way?
+* Myles: From the legal standpoint there will be clear guidelines for participation as you graduate to different levels. We have a vision of what a sustainable open governance model looks like, and if you want to join through our process, we can guide you through that process.
+* Dave: When you get to legal issues it gets a bit trickier, but if you look at the JSF lists for example - we do have a membership process. The period of time that you’re not a voting member could be very short if you want to be active and be a member of CPC. You can follow a best practice to get there.
+* Jory: My Concerns for these definitions is that you might be precluding these smaller projects from having a voice and being able to take on voting decisions.
+* Matteo: For me it seems a little bit complicated. If a project isn’t very active, would they be willing to invest time in building the foundation?
+* Jory: There’s a difference between thriving and small projects. There are upper limits to projects, though they are thriving, and that’s ok.
+* Michael: We’re trying to word it so anyone can participate in the regular work. There are multiple open paths to becoming voting members, if you’re big enough - get it, but if you’re smaller, you can get recognition as voting members by the many paths we have to get there.
```

Comment:
I would change it to:

* Michael: We’re trying to word it so anyone can participate in the regular work. There are multiple open paths to becoming voting members, if the project is big enough there is direct representation. If the project is smaller there are still several paths to get there in the governance being proposed.

Reply:
done! thanks!